flow/designs/tsmc65lp/swerv_wrapper/metadata-hier_rtlmp-ok.json

```diff
@@ -90,20 +90,20 @@
   "globalroute__timing__drv__max_cap": 497.0,
   "globalroute__timing__clock__slack": "N/A",
   "globalroute__timing__clock__period": "N/A",
-  "globalroute__runtime__total": "1:06.49",
-  "globalroute__cpu__total": 65.35,
-  "globalroute__mem__peak": 1362884.0,
-  "detailedroute__route__wirelength": 6173289.0,
-  "detailedroute__route__vias": 879114.0,
+  "globalroute__runtime__total": "1:30.41",
+  "globalroute__cpu__total": 88.49,
+  "globalroute__mem__peak": 1362164.0,
+  "detailedroute__route__wirelength": 6174805.0,
+  "detailedroute__route__vias": 879321.0,
   "detailedroute__route__drc_errors": 0,
-  "detailedroute__runtime__total": "19:41.55",
-  "detailedroute__cpu__total": 6216.57,
-  "detailedroute__mem__peak": 8268440.0,
+  "detailedroute__runtime__total": "18:52.89",
+  "detailedroute__cpu__total": 5956.91,
+  "detailedroute__mem__peak": 8545596.0,
   "finish__clock__latency__min": 1.16,
   "finish__clock__latency__max": 1.47,
   "finish__clock__skew__worst": 0.31,
   "finish__timing__setup__tns": 0.0,
-  "finish__timing__setup__ws": 0.59,
+  "finish__timing__setup__ws": 0.58,
```

Comment:
should we tighten more?

Reply:
As mentioned we will tighten the SDC constraints incrementally once the pipeline is running "green" nightly.